Formula/scipoptsuite.rb

```diff
@@ -0,0 +1,62 @@
+class Scipoptsuite < Formula
+  desc "Mixed Integer Programming (MIP) solver and Branch-and-Cut-and-Price Framework"
+  homepage "https://scipopt.org/"
+  url "https://www.scipopt.org/download/release/scipoptsuite-8.0.3.tgz"
+  sha256 "5ad50eb42254c825d96f5747d8f3568dcbff0284dfbd1a727910c5a7c2899091"
+  license all_of: ["Apache-2.0", "LGPL-3.0-or-later"]
+
+  depends_on "cmake" => :build
+  depends_on "bison"
+  depends_on "boost"
+  depends_on "cppad"
+  depends_on "flex"
+  depends_on "gmp"
+  depends_on "gnuplot"
+  depends_on "ipopt"
+  depends_on "tbb"
+  depends_on "zlib"
+
+  def install
+    cmake_args = std_cmake_args + %w[
+      -DPARASCIP=ON
+      -DPAPILO=ON
+      -DSOPLEX=ON
+      -DGCG=ON
+      -DZIMPL=ON
+      -DBOOST=ON
+      -DGMP=ON
+      -DQUADMATH=ON
+      -DIPOPT=ON
+      -DIPOPT_DIR="#{prefix}"
+      -DZLIB=ON
+      -DREADLINE=OFF
+      -DSYM=bliss
+      -DEXPRINT=cppad
+      -DCLIQUER=OFF
+    ]
+
+    system "cmake", "-B", "scipoptsuite-build", "-S", ".", *cmake_args
+    system "cmake", "--build", "scipoptsuite-build"
+    system "cmake", "--install", "scipoptsuite-build"
+  end
+
+  test do
+    system "cmake", "-B", "build", "-S", "scipoptsuite/scip/examples/Queens", "-DCMAKE_BUILD_TYPE=Release"
+    system "cmake", "--build", "build"
+    system "./build/queens", "5"
+
+    system "scip", "--version"
+
+    # Verifies that dependencies are properly linked
```

Comment:
Why not compile something that links against these?

Reply:
I used the same procedure that the team is using for testing the Scipoptsuite on conda.

Reply:
I don't know the rules for coda tests. But brew's rules state that a version check isn't enough

Reply:
Ok, thanks. Then I will work on the test.